doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<p>For this example as well as the ImageNet one, add that a GPU is recommended to run the example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>